console/billing/how-to/use-billing-alerts.mdx

```diff
@@ -0,0 +1,63 @@
+---
+meta:
+  title: How to use billing alerts
+  description: This page explains how to use billing alerts
+content:
+  h1: How to use billing alerts
+  paragraph: This page explains how to use billing alerts
+tags: billing-alert alerts billing budget 
+dates:
+  validation: 2024-02-14
+  posted: 2024-02-14
+categories:
+  - billing
+---
+
+The **billing alerts** functionality within the Scaleway console allows users to oversee and control their expenses efficiently. This feature allows you to create alerts that notify users when predefined budget thresholds are met. Your alerts can be seamlessly delivered through SMS, email, or API webhook channels.
+
+Your budget, denominated in euros, represents the limit for your expenses. Simultaneously, the threshold corresponds to a predetermined percentage of this budget. You have the autonomy to define both the budget and the threshold within the console interface.
+
+**For example:**
+
+  - If you have defined a budget of €1000, you may configure an email notification to be sent once you have consumed 50% of this limit. In this case, once your consumption gets to €500 you will receive an email informing you the limit threshold has been reached.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+</Message>
+
+
+1. Go to the Billing page by clicking on **Billing** in the Organization drop-down menu of the Scaleway console.
+2. Click **Consumption**.
+3. Scroll down to find **Billing alerts**.
+    <Message type="important">
+      Billing alerts only provide a rough estimate of what may be charged to your monthly invoice. Additional costs related to your usage before the alert is triggered may still appear on your invoice after the alert is generated. Only your monthly invoice is considered legally valid.
```

Comment:
I'd rather put this message at the end of the procedure

Reply:
> I'd rather put this message at the end of the procedure

You're right.. it's better to have this message at the end.